src/GuiBase/TreeModel/TreeModel.cpp

```diff
@@ -6,6 +6,8 @@
 namespace Ra {
 namespace GuiBase {
 
+using namespace Ra::Core::Utils;
```

Comment:
This should not be mandatory with the modification right?

Reply:
Still necessary for LogLevel.